src/utilities/forms.py

```diff
@@ -102,3 +104,42 @@ class Meta:
             # https://code.djangoproject.com/ticket/24453
             'sites': forms.MultipleHiddenInput(),
         }
+
+
+class MenuItemForm(forms.ModelForm):
+    url = forms.RegexField(
+        label=_("URL"),
```

Comment:
Why is this required?  the url field uses URLOrRelativeURLField which should already take care of this? If it doesn't then there's no point using URLOrRelativeURLField and we can just make it a standard Django URLField and remove the useless dependency.

Reply:
the regexfield is needed to properly validate relative URLs on the frontend, for a reason I'm currently unsure of, but the code used is sourced from django.contrib.flatpages.forms.FlatPageForm (another frontend field that accepts relative urls properly)
the decision to keep the URLOrRelativeURLField in model creation was made to maintain the same effect when creating menu items from admin.